binr/rabin2/Makefile

```diff
@@ -3,6 +3,7 @@ BINDEPS=r_magic r_core r_db r_bin r_egg r_cons
 BINDEPS+=r_config r_io r_flags r_asm
 BINDEPS+=r_debug r_hash r_lang r_anal r_parse r_bp r_reg
 BINDEPS+=r_search r_syscall r_diff r_socket r_fs r_magic r_util
+CLFAGS+=-ldl
```

Comment:
just use the r_lib api, you dont have to hardcode a system library here, because r_lib is shipped with r_util and this is already linked